### PR TITLE
feature/io-493_splash_screen

### PR DESCRIPTION
**Pull request by egor-slatvinskiy-idf**

### CODE REVIEW

File: packages/presentation/lib/my_app.dart [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+  @override
+  State<StatefulWidget> createState() => _MyAppState();
+}
+
+class _MyAppState extends BlocScreenState<StatefulWidget, AppBloc> {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      color: const Color.fromRGBO(15, 27, 43, 1),
```
Comment (alexei-kruk-idf) [inline packages/presentation/lib/my_app.dart]: you can create class abstract or singleton AppColors and move color there

File: packages/presentation/lib/ui/home_page/home_widget.dart [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  int _selectedTab = 0;
+
+  void onSelectedTab(int index) {
+    if (_selectedTab == index) return;
+    setState(
+      () {
+        _selectedTab = index;
+      },
+    );
+  }
```
Comment (alexei-kruk-idf) [inline packages/presentation/lib/ui/home_page/home_widget.dart:32]: move to bloc

File: packages/presentation/lib/ui/splash_screen/splash_widget.dart [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:presentation/base/bloc_screen.dart';
+import 'package:presentation/navigation/base_page.dart';
+import 'package:presentation/ui/splash_screen/bloc/splash_bloc.dart';
+
+const image = 'packages/presentation/images/Frame.png';
+
+class SplashWidget extends StatefulWidget {
```
Comment (alexei-kruk-idf) [inline packages/presentation/lib/ui/splash_screen/splash_widget.dart:9]: better rename  SplashScreen and other same

File: packages/presentation/pubspec.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+     - images/
+  # fonts:
+  #   - family: Schyler
+  #     fonts:
+  #       - asset: fonts/Schyler-Regular.ttf
+  #         style: italic
+  #   - family: Trajan Pro
+  #     fonts:
+  #       - asset: fonts/TrajanPro.ttf
+  #         weight: 700
```
Comment (alexei-kruk-idf) [inline packages/presentation/pubspec.yaml]: remove

File: pubspec.lock [hunk 5]
```diff
@@ -50,6 +50,20 @@ packages:
...
+    source: path
+    version: "0.0.1"
+  domain:
+    dependency: transitive
+    description:
+      path: "packages/domain"
+      relative: true
+    source: path
+    version: "0.0.1"
   fake_async:
```
Comment (alexei-kruk-idf) [inline pubspec.lock]: gitignore

**Review by alexei-kruk-idf — APPROVED** [review]

**Review by vladimir-hodaryonok-idf — APPROVED** [review]

**Review by elizaveta-seleznyova-idf — APPROVED** [review]

**Review by sergei-mikhailovskii-idf — APPROVED** [review]

File: packages/presentation/lib/ui/splash_screen/splash_widget.dart (hunk 3 above)
Comment (egor-slatvinskiy-idf) [reply packages/presentation/lib/ui/splash_screen/splash_widget.dart:9]: I call widgets, widgets) it's easier for me to navigate

**Pull request merged**